openmdao/test/sellar.py

```diff
@@ -142,8 +142,10 @@ def __init__(self):
         self.add('px', IndepVarComp('x', 1.0), promotes=['*'])
         self.add('pz', IndepVarComp('z', np.array([5.0, 2.0])), promotes=['*'])
 
-        self.add('d1', SellarDis1withDerivatives(), promotes=['*'])
-        self.add('d2', SellarDis2withDerivatives(), promotes=['*'])
+        sub = self.add('sub', Group(), promotes=['*'])
+        sub.ln_solver = ScipyGMRES()
+        sub.add('d1', SellarDis1withDerivatives(), promotes=['*'])
+        sub.add('d2', SellarDis2withDerivatives(), promotes=['*'])
```

Comment:
OK, I put SellarDerivatives back to its old configuration with the gmres at the top.